### ISSUE TITLE
api.Performance.memory - android webview doesn't work properly

### ISSUE THREAD

**Issue by archo5** [mdn/browser-compat-data#16560]
### What type of issue is this?

Browser bug (a bug with a feature that may impact site compatibility)

### What information was incorrect, unhelpful, or incomplete?

docs say that Android WebView provides this API but it seems to only return `usedJSHeapSize = 10000000`

### What browsers does this problem apply to, if applicable?

Chromium (Chrome, Edge 79+, Opera, Samsung Internet)

### What did you expect to see?

usedJSHeapSize exceeding ~32MB~ 64MB (see code below) - which I've been able to get in Opera on PC

### Did you test this? If so, how?

ran the html attached below, got this (line numbers are wrong since the html gets some stuff prepended during download):

```
I/cr_LibraryLoader: Loaded native library version number "87.0.4280.101"
...
I/chromium: [INFO:CONSOLE(33)] "leak created", source: https://*redacted*/highmem.htm?v=1 (33)
I/chromium: [INFO:CONSOLE(34)] "LLLLLLLLLLLLLLLLLLLLLLLLLLLL*removed further Ls*
I/chromium: [INFO:CONSOLE(35)] "usedJSHeapSize:10000000", source: https://*redacted*/highmem.htm?v=1 (35)
W/Looper: Slow Looper main: doFrame is 392ms late because of 1 msg
```

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

🤷‍♀️

### Do you have anything more you want to share?

`highmem.htm`:
```html
<font color="#F00">leaking 32 MB of memory...</font>
<script type="text/javascript">
window.leak = "L".repeat(1024 * 1024 * 32)
console.log("leak created")
console.log(window.leak)
console.error("usedJSHeapSize:" + window.performance.memory.usedJSHeapSize);
</script>
```

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Performance/memory

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `api.Performance.memory`
* Report started: 2022-06-06T02:15:43.790Z

</details>

**Comment by archo5** [comment]
hmm, it actually seems like there might be some weird bug instead

I've pasted the full code below (after moving the inserted code to source):

````html
<script>
if (window.performance.memory)
{
	setTimeout(function()
	{
		console.error("PERF:" + window.performance.memory.usedJSHeapSize);
	},
	1000);
}

console.info("initialized");
</script>
<font color="#F00">leaking 32 MB of memory...</font>
<script type="text/javascript">
window.leak = "L".repeat(1024 * 1024 * 32)
console.log("leak created")
console.error("usedJSHeapSize:" + window.performance.memory.usedJSHeapSize);
</script>
````

the output (PC Chromium):

```
initialized
highmem2.htm:16 leak created
highmem2.htm:17 usedJSHeapSize:10000000
(anonymous) @ highmem2.htm:17
highmem2.htm:6 PERF:10000000
(anonymous) @ highmem2.htm:6
setTimeout (async)
(anonymous) @ highmem2.htm:4
```

I initially thought some of the changes resulted in this but apparently even the original code loaded later can trigger this 🤔

not entirely sure what's causing all these issues exactly but it's probably worth exploring further and documenting

the bottom line is that the API doesn't seem to be working consistently

**Comment by archo5** [comment]
possibly relevant: https://stackoverflow.com/questions/25389096/why-is-memory-usage-not-correctly-updated
it seems possible that the cache timeout got increased again at some point 🤔

**Comment by Elchi3** [comment]
I think we can close this with https://github.com/mdn/content/pull/22476. performance.memory is deprecated and unreliable. Use measureUserAgentSpecificMemory (https://wicg.github.io/performance-measure-memory/) instead.
